real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java

```diff
@@ -46,10 +50,6 @@
 	/** Steering Angle offset?? */
 	private final int OFFSET = -200;
 
-	//Stored commanded values
-	private short commandedAngle = 0;
-	private boolean commandedBrakeEngaged = true;
-
 	// accumulated
 	private int encTicks = 0;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 'potValue' could be replaced by a local variable.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1770851711/issues/source?bid=2938665&fileBranchId=3059924#l55)